TheWarWithin/RogueSubtlety.lua

```diff
@@ -1459,7 +1451,7 @@ spec:RegisterAbilities( {
     symbols_of_death = {
         id = 212283,
         cast = 0,
-        charges = 1,
+        charges = function() return 1 + talent.death_perception.rank end,
```

Comment:
For `charges` and `recharge`, I generally prefer to wrap in an `if` statement so that it ends up being `nil` if the condition isn't met (it helps a little bit with counting cooldowns vs. recharges).

`charges = function() return if talent.death_perception.enabled then return 2 end end,`
`recharge = function() return if talent.death_perception.enabled then return 30 end end,`

This way, when Death Perception isn't talented, the addon will ignore that there are charges and just treat it as a single cooldown.

Reply:
Oops, needed more caffeine, I knew this.